chrome/content/zotero/integration/citationDialog.js

```diff
@@ -645,7 +645,7 @@ class LibraryLayout extends Layout {
 			},
 			isSearch: () => true,
 			isSearchMode: () => true,
-			setSearch: str => collectionTreeRow.setSearch(str),
+			setSearch: (type, str) => collectionTreeRow.setSearch(type, str),
```

Comment:
Wouldn't matter, but wrong order. Should be `(searchText, mode)`.

Reply:
Right, thank you. I swapped them in the repushed commit.